php/Exchange.php

```diff
@@ -132,6 +132,7 @@ class Exchange {
     public $currencies = array();
     public $base_currencies = null;
     public $quote_currencies = null;
+    public $generatedNetworkData = array();
```

Comment:
this is not needed right?

Reply:
it's needed in upcoming PR which depends on this PR, but ok, i'll remove from here.